src/Book/CreateOrEditBook.css

```diff
@@ -206,3 +173,7 @@
 button:hover {
     cursor: pointer;
 }
+
+.creat-or-edit-tags-container {
+    width: 45%;
+}
```

Comment:
Notice that the rows on the form are not well aligned this is happening because you assign the `margin: 2%` to the tag-container and not to this main container